src/services/message-service.ts

```diff
@@ -3,17 +3,26 @@ import {
   IEnterQueueMessage,
   ISocketMessage,
   ITextMessage,
+  ISocketFile,
+  IStudent,
+  IChat,
   IFile,
 } from '../interfaces';
 import { MESSAGE_TYPES } from '../config';
 import { IReconnectMessage } from '../interfaces/IReconnectMessage';
+import { IJoin } from '../interfaces/IJoin';
```

Comment:
```suggestion
import { IJoin, IReconnectMessage } from '../interfaces';
```
Slå sammen linje 12 og 13